src/mesh.rs

```diff
@@ -102,8 +103,19 @@ pub fn load_mesh(
     let file_string = filename.as_ref();
 
     #[cfg(feature = "assimp")]
-    if use_assimp && !file_string.starts_with("https://") && !file_string.starts_with("http://") {
-        return load_mesh_assimp(file_string, scale, opt_color, group, use_texture);
+    if use_assimp {
+        let is_url = file_string.starts_with("https://") || file_string.starts_with("http://");
+        let buf = if is_url {
+            crate::utils::fetch_read(file_string)?
+        } else {
+            use std::io::Read;
+            let mut file = std::fs::File::open(file_string)?;
+            let mut buf = vec![];
+            file.read_to_end(&mut buf)?;
+            buf
+        };
+        let data = String::from_utf8(buf)?;
```

Comment:
Some meshes are not strings (e.g., binary STL), so I think this would not work well with them.